src/SamplesApp/UITests.Shared/Windows_UI_Xaml/xBindTests/NoPhaseBinding_Large.xaml

```diff
@@ -420,7 +420,6 @@
 										<ContentPresenter x:Name="ContentPresenter"
 															ContentTransitions="{TemplateBinding ContentTransitions}"
 															ContentTemplate="{TemplateBinding ContentTemplate}"
-															not_win:ContentTemplateSelector="{TemplateBinding ContentTemplateSelector}"
 															Content="{TemplateBinding Content}"
```

Comment:
`ContentTemplate` and `Content` should be also removed here, no?

Reply:
That one seems to be necessary apparently (and is also not marked `not_win`).